src/components/ToolIcon.scss

```diff
@@ -153,6 +154,8 @@
     }
   }
 
+  // move the lock button out of the way on small viewports
+  // it begins to collide with the GitHub icon before we switch to mobile mode
   @media (max-width: 760px) {
```

Comment:
can we get those in variables? 425/760

Reply:
Since they’re only used once and are fairly specific to the context I think it’s ok to use them as-is (with the comment that I added) and only pull them out if it becomes a problem